grpc/pom.xml

```diff
@@ -20,24 +20,23 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <parent>
-    <artifactId>helidon-project</artifactId>
-    <groupId>io.helidon</groupId>
-    <version>1.1.0-SNAPSHOT</version>
-  </parent>
-
-  <groupId>io.helidon.grpc</groupId>
-  <artifactId>helidon-grpc-project</artifactId>
-  <name>Helidon gRPC Project</name>
-  <packaging>pom</packaging>
-
-  <modules>
-    <module>core</module>
-    <module>server</module>
-    <module>client</module>
-    <module>metrics</module>
-  </modules>
-
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
Same as before - why is this marked as modified?

Reply:
@tomas-langer Because some of the gRPC poms were using inconsistent indentation (mix of 2 spaces and 4 spaces indents).

Reply:
and once again, ok ;)